src/model/transaction/__tests__/insertFragmentIntoContentState-test.js

```diff
@@ -107,3 +118,277 @@ test('must apply multiblock fragments', () => {
     ]),
   );
 });
+
+test('must apply a single ContentBlockNode from fragment', () => {
```

Comment:
Lets rename this to something more meaningful like, 

`must be able to insert a fragment with a single ContentBlockNode`